tests/single-node/test.sh

```diff
@@ -35,6 +35,10 @@ setup_suite() {
         make_shell kubectl get nodes
 }
 
+test_deploy_again() {
+        make_shell ansible-playbook -i "$(pwd)/inventory" metal-k8s.yml --skip elasticsearch || die
```

Comment:
Should this be `die` or use an `assert`?

Reply:
Is not the same effect ?

Reply:
No. `die` will stop execution of tests altogether (it's `exit 1`). One of the *bash_unit* `assert`s will make the one test-case fail, but continue running others.